### PR TITLE
Add ID

### PR DESCRIPTION
**Pull request by ArvindHarinder1**

### CODE REVIEW

**Comment by ArvindHarinder1** [comment]
@Lauragra could you please review/ approve

File: api-reference/beta/resources/synchronization-synchronizationschema.md [hunk 1]
```diff
@@ -45,41 +45,35 @@ The following sections describe the high-level components of the synchronization
 
 | Property      | Type      | Description    |
 |:--------------|:----------|:---------------|
-|directories            |[directoryDefinition](synchronization-directorydefinition.md) collection   |Describes directories and objects that are part of the [synchronizationJob](synchronization-synchronizationjob.md) or [synchronizationTemplate](synchronization-synchronizationtemplate.md). |
-|synchronizationRules   |[synchronizationRule](synchronization-synchronizationrule.md) collection   |A collection of synchronization rules configured for the [synchronizationJob](synchronization-synchronizationjob.md) or [synchronizationTemplate](synchronization-synchronizationtemplate.md), |
+|id|String|Uniquely identifier for the schema.|
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-synchronizationschema.md:48]: Should say "Unique identifier for the schema."

File: api-reference/beta/resources/synchronization-synchronizationschema.md [hunk 2]
```diff
@@ -45,41 +45,35 @@ The following sections describe the high-level components of the synchronization
...
+|synchronizationRules   |[synchronizationRule](synchronization-synchronizationrule.md) collection   |A collection of synchronization rules configured for the [synchronizationJob](synchronization-synchronizationjob.md) or [synchronizationTemplate](synchronization-synchronizationtemplate.md). |
 |version                |String                             |The version of the schema, updated automatically with every schema change.|
 
 
-## JSON representation
+## Relationships
+|Relationship|Type|Description|
+|:---|:---|:---|
+|directories|[directoryDefinition](../resources/synchronization-directorydefinition.md) collection|Represents directory and its objects.|
```
Comment (davidmu1) [inline api-reference/beta/resources/synchronization-synchronizationschema.md:56]: Contains the collection of directories and all of their objects.

**Comment by ArvindHarinder1** [comment]
@Lauragra / @CelesteDG  could you please merge?

**Comment by Lauragra** [comment]
@ArvindHarinder1 , in the future, please set the *ready to merge* label to signal that a PR is ready for publishing.

**Comment by ArvindHarinder1** [comment]
> @ArvindHarinder1 , in the future, please set the _ready to merge_ label to signal that a PR is ready for publishing.

Will do, thanks for the heads up.

**Pull request merged**